src/react-components/ui-root.js

```diff
@@ -1414,6 +1424,11 @@ class UIRoot extends Component {
     // Allow scene picker pre-entry, otherwise wait until entry
     const showMediaBrowser = mediaSource && (mediaSource === "scenes" || this.state.entered);
 
+    const discordBridges = this.discordBridges();
+    const messageEntryPlaceholder = discordBridges
+      ? `Send to hub (+ ${discordBridges.map(ch => "#" + ch).join(", ")})...`
+      : "Send to hub...";
```

Comment:
We don't use the term `hub`, so should be `room` as before. Also, my 2c is the message should be:

`Send to room and #foo...`

Reply:
Oops, I can never remember which we use.